implementation/test/TBTCSystemTest.js

```diff
@@ -67,6 +82,16 @@ describe("TBTCSystem", async function() {
         "TBTCSystem did not correctly forward value to keep factory",
       ).to.eq.BN(openKeepFee)
     })
+
+    it("reverts if caller does not match a valid TDT", async () => {
```

Comment:
`reverts if the caller is not a deposit`?

Reply:
This is how I had it initially, but it didn't really feel accurate. We aren't really checking that it is a Deposit, we are checking that it is a TDT. Also in the test we just mint the token 🤷‍♂ 

I can go either way